src/app/(admin)/new_dashboard/guides/[id]/page.tsx

```diff
@@ -50,13 +48,17 @@ const GuidePage: React.FC = () => {
     }
 
     return (
-      <PageContainer title={guide.title}>
-        <DashboardCard title={guide.title} subtitle={creatorAndDateInfo}>
-          <div className="quill-content">{parse(guide.contentHTML)}</div>
-        </DashboardCard>
+      <Box>
+        <Typography variant="h1">{guide.title}</Typography>
+        <Typography variant="subtitle1">{creatorAndDateInfo}</Typography>
+        <Box className="quill-content">
+          {parse(guide.contentHTML)}
+        </Box>
         <Divider sx={{ mt: 2, mb: 2, border: "none" }} />
         <ReviewList guideId={guide.id} reviews={guide.reviews ?? []} />
-      </PageContainer>
+        <Divider sx={{ mt: 5, mb: 5, border: "none" }} />
+        {selectedContact && <ChatPopup selectedContact={selectedContact} />}
+      </Box>
```

Comment:
Why would we need to insert the component in each page? 

https://en.wikipedia.org/wiki/Don%27t_repeat_yourself